src/main/java/com/oneops/secrets/command/OnConsole.java

```diff
@@ -0,0 +1,9 @@
+package com.oneops.secrets.command;
+
+import static com.oneops.secrets.utils.Color.warn;
+
+public abstract class OnConsole {
```

Comment:
Why do you need this abstract class ? I would make this changes simple and don't introduce unnecessary class hierarchy